Task3.py

```diff
@@ -0,0 +1,19 @@
+string = input()
+
+
+def decoratorHtml(left, right):
+    def decorator(function):
+        def wrapper(n1):
+            return left + function(n1) + right
+
+        return wrapper
+
+    return decorator
+
+
+@decoratorHtml("<html>", "</html>")
```

Comment:
a closing tag always corresponds to an opening tag, so you don't need to pass both, also there is no point to pass braces